src/Compilers/VisualBasic/Portable/Binding/Binder_Expressions.vb

```diff
@@ -251,7 +251,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                     ' e.g. SyntaxKind.MidExpression is handled elsewhere
                     ' NOTE: There were too many "else" cases to justify listing them explicitly and throwing on
                     ' anything unexpected.
-                    Debug.Assert(node.ContainsDiagnostics, String.Format("Unexpected {0} syntax does not have diagnostics", node.Kind))
+                    ' re-enable the assert once this issue is fixed
```

Comment:
Could you add PROTOTYPE(IOperation) to all the assertions that were disabled in this PR?
I think that we should not merge back to master with the relaxed assertions.